Emoji fixed styles and integration with input box

johnhorsema opened the pull request.

Commit:
Emoji support (there is some problem)

Anyone can help me with the css of the text area?

johnhorsema committed:
Resolve issue #5. Needs tidying up though.

johnhorsema committed:
Solved clear input bug.

koyip commented:
As we've changed to use "ng-bind-html", we need to sanitize the output to prevent HTML injection / XSS.

koyip commented:
And, question content does not show emoji images. Only title does.

koyip committed:
Merge branch 'gh-pages' into emoji

koyip committed:
Fixed all known issues related to the new emoji text area.
Escape HTML and render emoji images in the question title and content properly.
Refactored the way to add tag links.
Removed title input field temporarily.
Removed unused directives.

koyip commented:
I have fixed the issues above in the latest commit. Please check once again then merge.
